tests/file_loading.py

```diff
@@ -0,0 +1,44 @@
+"""
+File:           file_loading.py
+Description:    Constants and utilities used for loading files/recipes.
+"""
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Final
+
+from conda_recipe_manager.parser.recipe_parser import RecipeParser
+from conda_recipe_manager.parser.recipe_parser_convert import RecipeParserConvert
+
+# Path to supplementary files used in test cases
+TEST_FILES_PATH: Final[str] = "tests/test_aux_files"
+
+
+def load_file(file: Path | str) -> str:
+    """
+    Loads a file into a single string
+    :param file: Filename of the file to read
+    :returns: Text from the file
+    """
+    with open(Path(file), "r", encoding="utf-8") as f:
+        return f.read()
+
+
+def load_recipe(file_name: str) -> RecipeParser:
```

Comment:
if you prefer you can use factories as fixtures here and remove the ``__init__.py`` and this bit here, and put everything in the ``conftest.py``
example:
https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#factories-as-fixtures

Reply:
I appreciate and am aware, but I wanted to keep this PR smaller and focus refactoring one thing.